docs/README.md

```diff
@@ -577,11 +577,16 @@ onTransaction = transactionCallback
 
 <!-- tabs:end -->
 
-## on-open
+## on-open ( *optional* )
```

Comment:
Let's remove the `( optional )` bit here as it's unclear _what_ is optional to somebody reading this for the first time.